src/test/java/com/khospodarysko/SpinnersTest.java

```diff
@@ -11,16 +16,37 @@
     private static final Logger logger = LoggerFactory.getLogger(SpinnersTest.class);
 
     @Test
-    public void test() {
+    public void testSpinners() {
         driver.get("file://" + absoluteFilePath("spinners"));
+        WebElement spinnerSmall = driver.findElement(By.cssSelector(".spinner-small"));
+        WebElement spinnerLarge = driver.findElement(By.cssSelector(".spinner-large"));
 
-        logger.info("{}", areSpinnersLoaded());
+        logger.info("{}",
+        areSpinnersLoaded(driver, spinnerSmall));
+        areSpinnersLoaded(driver, spinnerLarge);
+
+        Assert.assertTrue(isElementNotDisplayed(spinnerSmall), "Small spinners are disappeared");
+        Assert.assertTrue(isElementNotDisplayed(spinnerLarge), "Large spinners are disappeared");
     }
 
     /**
      * @return true if all spinners have disappeared, false otherwise
      */
-    public boolean areSpinnersLoaded() {
-        return false;
+    public boolean areSpinnersLoaded(WebDriver driver, WebElement element) {
+        WebDriverWait wait = new WebDriverWait(driver, 20);
+        return wait.until(new ExpectedCondition<Boolean>() {
+            @Override
+            public Boolean apply(WebDriver driver) {
+                try {
+                    return !element.isDisplayed();
```

Comment:
Need to check this for all small and all large spinners, not only for single element.

Reply:
Can i use here operator AND - 
return !(smallSpinner.isEmpty()) && !(largeSpinner.isEmpty())